builtin/logical/pkiext/pkiext_binary/pki_cluster.go

```diff
@@ -104,6 +110,16 @@ func (vpc *VaultPkiCluster) AddNameToHostsFile(ip, hostname string, logConsumer
 	return nil
 }
 
+func (vpc *VaultPkiCluster) AddDNSRecord(hostname, recordType, ip string) error {
```

Comment:
Should we get rid of the recordType argument here? Or else if we do want it generic, recordType should be passed through to AddRecord and ip should be renamed to something like recordValue or value?

Reply:
Ah, I fixed that on the next PR, recordType should go down to the DNS call rather than it being hard-coded to "A" records.

Reply:
https://github.com/hashicorp/vault/pull/20486 was the one I was thinking of.